doc-content/enterprise-only/getting-started-kogito/ref-kogito-microservice-deploy-troubleshooting.adoc

```diff
@@ -0,0 +1,22 @@
+[id="ref-kogito-microservice-deploy-troubleshooting_{context}"]
+= {KOGITO} microservice deployment troubleshooting
+
+Use the information in this section to troubleshoot issues that you might encounter when using the {KOGITO} Operator to deploy {KOGITO} microservices. The following information is updated as new issues and workarounds are discovered.
+
+No builds are running::
+If you do not see any builds running nor any resources created in the relevant namespace, enter the following commands to retrieve running pods and to view the {KOGITO} Operator log for the pod:
```

Comment:
```suggestion
If you do not see any builds running nor any resources created in the relevant namespace, enter the following commands to retrieve running pods and to view the {KOGITO} Operator logs for the pod:
```

Reply:
nor is a bit archaic. It is acceptable and more common to use or.
log file
Heena is there more than one file?

Reply:
Yes